### ISSUE TITLE
kbox drush up, unable to untar now 10.6

### ISSUE THREAD

**Issue by labboy0276** [kalabox/kalabox#740]
Hello!

I went in to update a site we be working on via kbox drush up and I get a whole bunch of untar errors now as soon below:

```
Do you really want to continue with the update process? (y/n): y
Unable to untar ./sites/all/modules/contrib/cleaner-7.x-1.3.tar.gz.  [error]
Project cleaner was updated successfully. Installed version is now 7.x-1.3.
Backups were saved into the directory                                [ok]
/root/drush-backups/pantheon/20151123145433/modules/cleaner.
Unable to untar ./sites/all/modules/contrib/ctools-7.x-1.9.tar.gz.   [error]
Project ctools was updated successfully. Installed version is now 7.x-1.9.
Backups were saved into the directory                                [ok]
```

**Comment by pirog** [comment]
were you able to get the new updates even with the error?

**Comment by labboy0276** [comment]
Negative, after it went through all that, it reverts all the updates

**Comment by pirog** [comment]
good to know. will drop this into the backlog

**Comment by jpsalter** [comment]
Same problem on my setup 0.10.6

**Comment by pirog** [comment]
This should be resolved in the 0.11.0 release on Friday
